internal/ocm/client.go

```diff
@@ -93,8 +108,15 @@ func (c *Client) do(
 		resBody = bytes.NewBuffer(j)
 	}
 
-	// Request
-	httpReq, err := http.NewRequestWithContext(ctx, httpMethod, reqURL.String(), resBody)
+	// processing the URL parameters map into a string
+	var paramsString strings.Builder
+	paramsString.WriteString("?")
+	for key, value := range params {
+		paramsString.WriteString(fmt.Sprintf("%s=%s,", key, value))
+	}
+	fullUrl := reqURL.String() + paramsString.String()
```

Comment:
Should be replaced with `url.Values` to create the URL query:
https://pkg.go.dev/net/url@go1.17.6#URL.Query
https://pkg.go.dev/net/url@go1.17.6#Values

Current implementation is missing `&` character for query parameters 2...n

Reply:
good catch.
yeah, I added a comma instead :)
fixing.